source/documentation/team-dashboard.md

```diff
@@ -0,0 +1,72 @@
+# Team Dashboard
+
+We now have dashboards!
+
+We currently display dashboards on our TV monitors (two of them), running on ChromeOS. This section 
+outlines what's currently displayed on them.
+
+## Technical
+
+The top monitor has technical dashboards worth keeping an eye on throughout the day.
+
+### Grafana
+
+Grafana is an open source analytics and monitoring platform, and we use it for monitoring the health 
+of GovWifi in realtime.
+
+In the past, GovWifi used to be part of a shared organisation on Grafana. Now, GovWifi has its own
+organisation, and we have admin-level access so we can now make our own changes without having to 
+always involve RE. 
+
```

Comment:
Since we have both dashboards under our org now, we can remove this paragraph.

Reply:
I wanted to have it here just to make it clear that we can now make changes ourselves, without having the need to ask the question whether we need RE (which was my question when I started on this piece of work). But I agree we can remove the info about shared org. What do you think?

Reply:
:+1: